src/transpile/fusion.hpp

```diff
@@ -432,7 +563,9 @@ cmatrix_t Fusion::matrix(const op_t& op) const {
   } else if (op.type == optype_t::matrix) {
     return op.mats[0];
   } else {
-    throw std::runtime_error("Fusion: unexpected operation type");
+    std::stringstream msg;
+    msg << "Fusion: unexpected operation type:" << op.type;
+    throw std::runtime_error(msg.str());
```

Comment:
```c++
 throw std::runtime_error("Fusion: unexpected operation type:" + std::to_string(op.type));
```

Reply:
I have changed codes as you pointed. Thanks.